docs/_embed/partner_api.md

```diff
@@ -140,11 +140,15 @@ The idea is to use an intermediate page that reads the access token from the URL
 
 Available parameters to the Apps resource:
 
-| parameter     | requirement | notes                                                                                  |
-| ------------- | ----------- | -------------------------------------------------------------------------------------- |
-| **client_id** | Required    | Your application client ID.                                                            |
-| **per_page**  | Optional    | (defaults to 100, max of 100) Limit the number of apps returned.                       |
-| **page**      | Optional    | (defaults to 1) The page number. Page number 1 refers to the first page in the result. |
+| parameter                   | requirement | notes                                                                                         |
+| --------------------------- | ----------- | --------------------------------------------------------------------------------------------- |
+| **client_id**               | Required    | Your application client ID.                                                                   |
+| **category**                | Optional    | Filter the results by app category.                                                           |
+| **is_in_zap_template_with** | Optional    | Fetch apps that appear in a Zap Template with your app. |
```

Comment:
It might make it more clear there's no value expected for this parameter like all the others by denoting in the notes "No value required for this param." or something like that.

Reply:
What about `Fetch apps that appear in a Zap Template with your app (No value required).`?

Reply:
Fetch apps that appear in a Zap Template with your app. No value required.
or
Fetch apps that appear in a Zap Template with your app (no value required).

Minor formatting suggestions^